app/forms/schools/add_participant_form.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Schools
+  class AddParticipantForm
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+    include ActiveModel::Serialization
+
+    TYPE_OPTIONS = {
+      ect: "Early Career Teacher",
+      mentor: "Mentor",
+    }.freeze
```

Comment:
Possible duplicate of `user.user_description` although I'm not 100% sure what it's actually doing

Reply:
`User#user_description` is an abomination and should be killed with fire! :D

But actually it returns much more options than what we need here, including admin, or lead provider users (have i mentioned that User knows waaaaay too much about the application already... :P)

Reply:
My main concern around this is that defining user definitions from keys in multiple places leads to possible variance - in this case it's just the capitalisation, in the future it could be more.

Reply:
So this constant does not define possible user types but rather possible options on the form. Ideally, I'd had those types translated in a single translation scope (and that will be dona later if time allows).